recipes/sphericart/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "sphericart" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/sphericart-{{ version }}.tar.gz
+    sha256: 2f3d2672a20efd7cf400cce42d0648f04d16fe8abaf834d167a1b45ef2f87e46
+
+build:
+  number: 0
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version == "11.8"]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("cuda") }}
+    - {{ stdlib("c") }}
+    - cmake
+    - cuda-nvcc
+    - cuda-toolkit
+    - make
+    - ninja
+    - pip         # [build_platform != target_platform]
+    - pybind11    # [build_platform != target_platform]
+    - python      # [build_platform != target_platform]
+    - pytorch     # [build_platform != target_platform]
+    - setuptools  # [build_platform != target_platform]
+  host:
+    - cuda-version {{ cuda_compiler_version }}
+    - cuda-nvcc
+    - cuda-toolkit
+    - pip
+    - python
+    - pytorch
+    - setuptools
+  run:
+    - _openmp_mutex  # [not osx]
```

Comment:
Instead of adding this, you probably need to add `libgomp/llvm-openmp` to both host and build.